packages/terra-clinical-onset-picker/src/OnsetPicker.scss

```
@@ -9,7 +9,33 @@
   }
 
   .field-inline {
-    margin-bottom: 0.357em;
-    margin-right: 0.357em;
+    margin-bottom: var(--terra-onset-picker-field-margin-bottom, 0.357em);
```

let's add the `clinical` namespace to the themeable variables.
```suggestion
    margin-bottom: var(--terra-clinical-onset-picker-field-margin-bottom, 0.357em);
```

---

Added clinical namespace.